### PR TITLE
New Neutral Quirk to indicate to Medical you're willing to be borged

### PR DESCRIPTION
**Pull request by zxaber**
## About The Pull Request
Adds a quirk that gives the user a "Cyborg Pre-approved" dogtag upon gaining, or (more usually) spawning with,  the quirk. Wearing this dog tag is to be considered granting full permission to be borged.
## Why It's Good For The Game
Waiting for medical to spend 15 minutes running tend wounds on your dead body, not to mention having to deal with your organ damage if you've been dead for a while, kinda sucks. It sucks more when you're waiting and just watching your stuff walk out the door. And if you're a Plasma man, you might as well go take a nap. Meanwhile, the rules get a bit eh about when you're allowed to just start throwing bodies at robotics. Not everyone appreciates being shifted to an entirely new role as a cyborg, but some players don't mind. 

I appreciate all you do medical. But if my body is more than a five minute fix to being revived, just borg me fam. You got other people you could be working on.

This isn't supposed to be mandatory in the opposite direction, mind. I intend that medical can still revive people with the tag, at their own discretion. 
## Changelog
:cl:
add: Added a new Neutral Quirk to indicate to Medical you're willing to be borged
/:cl:

### CODE REVIEW

File: code/datums/quirks/neutral_quirks/borg_ready.dm [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/datum/quirk/item_quirk/borg_ready
+	name = "Cyborg Pre-screened dogtag"
+	desc = "Get pre-approved for NT's experimental Cyborg program, with a dog tag that lets medical staff know."
+	icon = FA_ICON_ROBOT
+	value = 0
+	gain_text = span_notice("You hear a distant echo of beeps and buzzes.")
+	lose_text = span_danger("The distant beeping halts.")
+	medical_record_text = "Patient appears ready to abandon the flesh." //considering this quirk's whole point is a dogtag for medical to see, this feels redundant and silly
```
Comment (JohnFulpWillard) [inline code/datums/quirks/neutral_quirks/borg_ready.dm:8]:
```suggestion
	medical_record_text = "Patient is a registered brain donor for Robotics research."
```
Organ donors are a thing so maybe we can mimic that instead

**Comment by EricZilla** [comment]
This would be really nice to have on crazy rounds.

**Comment by ATH1909** [comment]
lowkey, I'd prefer for this to be opt-out rather than opt-in, but I might be crazy/weird

**Comment by SomeoneYouProbablyKnow** [comment]
Time to put that dogtag on my assignation target hehehe

**Comment by san7890** [comment]
you're going to need to get your GBP up playa

**Pull request merged**